feature/danggn/src/main/java/com/mashup/feature/danggn/ranking/DanggnRankingContent.kt

```diff
@@ -323,14 +358,16 @@ private fun RankingContent(
                 text = when (item) {
                     is DanggnRankingViewModel.RankingUiState.Ranking -> item.memberName
                     is DanggnRankingViewModel.RankingUiState.EmptyRanking -> "아직 ${index + 1}위가 없어요"
+                    is DanggnRankingViewModel.RankingUiState.PlatformRanking -> item.platformName
+                    is DanggnRankingViewModel.RankingUiState.MyRanking -> ""
                 },
                 style = SubTitle1.copy(
                     brush = Brush.linearGradient(
                         when (index) { // 반드시 2개 이상의 컬러가 필요해서 Gray900 넣어줬습니다
```

Comment:
```suggestion
                        when (index) { 
```

이제 주석은 지워도 되겠어요! 혹은 위치를 변경해야겠네요

Reply:
[c6baf52](https://github.com/mash-up-kr/mashup_Android/pull/314/commits/c6baf52348ac9aedd620f5c01b9309cc84f4d936)